contracts/interfaces/capital-protocol/IDistributor.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {IERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+
+import {IReferrer} from "./IReferrer.sol";
+
+interface IDistributor is IERC165 {
+    /**
+     * @dev Yield strategy.
+     * NONE - for tokens without yield strategy.
+     * NO_YIELD - for tokens without yield.
+     * AAVE - fot Aave yield strategy.
+     */
+    enum Strategy {
+        NONE,
+        NO_YIELD,
+        AAVE
+    }
+
+    /**
+     * @param depositPoolAddress The `DepositPool` contract address.
+     * @param token The yield token.
+     * @param chainLinkPath The path for `ChainLinkDataConsumer`.
+     * @param tokenPrice The token price, filled by the contract.
+     * @param deposited The deposited yield token amount.
+     * @param lastUnderlyingBalance The last calculate balance after thr yield.
+     * @param strategy The `Strategy`.
+     * @param aToken The `token` address after the deposit to Aave.
+     */
+    struct DepositPool {
+        address depositPoolAddress;
```

Comment:
Can this field be removed? 
I think you can add some zero state to Strategy enum that identifies the pool absence.